cmd/nvidia-cdi-hook/disable-device-node-modification/disable-device-node-modification.go

```diff
@@ -67,11 +68,11 @@ func NewCommand(logger logger.Interface) *cli.Command {
 	return &c
 }
 
-func validateFlags(c *cli.Context, cfg *options) error {
+func validateFlags(c *cli.Command, cfg *options) error {
 	return nil
 }
 
-func run(_ *cli.Context, cfg *options) error {
+func run(ctx context.Context, cmd *cli.Command, cfg *options) error {
```

Comment:
```suggestion
func run(_ *cli.Command, cfg *options) error {
```

Reply:
Not critical, but was there a reason for not just replacing cli.Context with cli.Command?